SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -94,20 +97,16 @@ void ASpatialFunctionalTest::OnAuthorityGained()
 
 void ASpatialFunctionalTest::RegisterAutoDestroyActor(AActor* ActorToAutoDestroy)
 {
-	if (HasAuthority())
+	if (ActorToAutoDestroy->HasAuthority())
 	{
 		Super::RegisterAutoDestroyActor(ActorToAutoDestroy);
-	}
-	else if(LocalFlowController != nullptr)
-	{
-		if(LocalFlowController->ControllerType == ESpatialFunctionalTestFlowControllerType::Server)
-		{
-			CrossServerRegisterAutoDestroyActor(ActorToAutoDestroy);
-		}
-		else
-		{
-			ServerRegisterAutoDestroyActor(ActorToAutoDestroy);
-		}
```

Comment:
Why was this else removed? If you call on a Worker that doesn't have authority over the Actor, it needs to forward it to whom does.

Reply:
Discussed this with Nuno and decided the use case should be to create and then register actor for auto destroy so should not need cross-server calls. Added an error log in case this is used incorrectly.